index.bs

```diff
@@ -197,6 +197,8 @@ interface BarcodeDetector {
 <dl class="domintro">
   <dt><dfn constructor for="BarcodeDetector"><code>BarcodeDetector(optional BarcodeDetectorOptions |barcodeDetectorOptions|)</code></dfn></dt>
   <dd>Constructs a new {{BarcodeDetector}} with the optional |barcodeDetectorOptions|, if present.
+    * If |barcodeDetectorOptions| is passed, and its |formats| are empty, then reject the Promise with a new {{DOMException}} whose name is {{TypeError}}, and abort any further steps.
```

Comment:
The `BarcodeDetector` constructor doesn't return a `Promise` so instead of rejecting you can directly "throw a `TypeError`." Note that `TypeError` is a separate class from `DOMException`.

Reply:
Should be fixed, thanks.